modules/standard/FileSystem.chpl

```diff
@@ -498,6 +498,16 @@ proc symlink(oldName: string, newName: string) {
   if err != ENOERR then ioerror(err, "in symlink " + oldName, newName);
 }
 
+/* Sets the file creation mask of the current process to mask, and returns
+   the previous value of the file creation mask.
+   mask: the file creation mask to use now.
+*/
+proc umask(mask: int): int {
```

Comment:
At least, it is described as doing the same thing.  I didn't look at its implementation.